apps/i18n/applab/en_us.json

```diff
@@ -355,6 +355,8 @@
   "makerSetupCheck": "Setup Check",
   "makerSetupCheckInternetConnection": "Please make sure you are connected to the internet, and [https://downloads.code.org](https://downloads.code.org/index.html) is reachable from your network.",
   "makerSetupChromebookPage": "Open this page ([{makerSetupPage}]({makerSetupPage})) on your Chromebook.",
+  "makerSetupCircuitPlaygroundDescription": "! The Adafruit [Circuit Playground Express](https://www.adafruit.com/product/3399) (and the older [Circuit Playground Classic](https://www.adafruit.com/product/3000)) is a programmable circuit board with built-in components that make getting started with physical computing quick and fun.\n\nIn unit 6 of [Computer Science Discoveries](https://code.org/educate/csd) students use the Circuit Playground tethered via USB to their computers to allow for online curriculum integration and interactive debugging of programs.\n\nPlease follow the instructions below to connect your board to Code.org.",
```

Comment:
why do all of the strings start with "! "?

Reply:
To test that translated strings are in fact being rendered.

Reply:
After testing, exclamation points were removed.